docs/getting-started.md

```diff
@@ -57,59 +65,73 @@ The [ort.yml][ort-yml] file will either contain the contents of the [.ort.yml][o
-- _Package details_: description, homepage, code repositories, binary and source artifact
+Click the '+' sign to the left of a package id, for example to the left of _NPM::acorn:7.1.1_, to see information such as:
+- _Details_: package description, homepage, code repositories, binary and source artifact
 - _Licenses_: declared licenses (from package metadata), detected licenses (found in source code), and concluded licenses (human overwrites)
 - _Paths_: dependency trail from root project to opened package id
 - _Scan Results_: copyright and licenses findings for files included in the source repository or artifact of opened package id
-- _Scope Excludes_: reasons why opened package id was marked as not included in released artifacts
+- _Scope Excludes_: (if present) reasons why the package you selected was marked as not included in released artifacts
+
+Note that when rule violations (_Rule Violations_) are displayed below _Summary_, the package-specific information includes _How to Fix_ in
+addition to the items shown above. _How to Fix_ shows instructions to help you correct a rule violation. 
 
 ### About Modal
 
-In the top-right corner of the Web App report you can find the _About_ button which, if clicked, shows you:
-- _Excludes_: contents of the [.ort.yml][ort-yml] file for the project. Not shown if no `.ort.yml` was found
-- _Labels_: labels were passed to the ORT _analyzer_ using one or more `-L` parameters. Not shown if no labels were set
-- _About_: details on the ORT project and render timestamp for the report
+At the right end of the top line in the Web App report -- the line that shows _Summary_, _Table and _Tree_ -- you will find an icon for the _About_ dialog. Click on it to see:
+- _Excludes_: contents of the [.ort.yml][ort-yml] file for the project -- not shown if no `.ort.yml` was found or is empty
+- _Labels_: labels that were passed to the ORT _analyzer_, using one or more `-L` parameters -- shown only if labels were set
+- _About_: details on the ORT project and the timestamp showing when the report was created
 
 ## Marking Files, Directories or Package Manager Scopes in your Project as not Included in Released Artifacts <a name="orthw-rc-excludes"></a>
 
-Not every file in the code repository of a software project is included in its release artifacts things such as build scripts, documentation or tests are often only used internally and are not released. You may want the treat these project internal files differently when reviewing license findings, e.g. a GPL-2.0-only licensed build script may be OK but GPL-2.0-only source code file which is included in the release artifacts may be not OK.
+It is possible that not every file in the code repository of a software project is included in the release artifacts. For example, build
+scripts, documentation or tests scripts are often only used internally and are not be part of public
```

Comment:
"are not be part" -> "are be part"

Would it make sense to modify the whole sentence to:

For example, build
scripts, documentation or tests scripts are often only used for development and do not end-up in the release artifact in any form.